DylibBinder/DBProperty.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using SwiftReflector;
+using SwiftReflector.Inventory;
+
+namespace DylibBinder {
+	internal class DBProperty : IAssociatedTypes {
+		public string Name { get; private set; }
+		public bool IsStatic { get; }
+		public string Type { get; }
+		public DBFunc Getter { get; }
+		public DBFunc Setter { get; }
+		public DBGenericParameters GenericParameters { get; }
+		public DBAssociatedTypes AssociatedTypes { get; } = new DBAssociatedTypes ();
+
+		public TypeAccessibility Accessibility { get; } = TypeAccessibility.Public;
+		public bool IsPossiblyIncomplete { get; } = false;
+		public bool IsDeprecated { get; } = false;
+		public bool IsUnavailable { get; } = false;
+		public bool IsOptional { get; } = false;
+		public Storage Storage { get; } = Storage.Addressed;
+
+		public DBProperty (PropertyContents propertyContents)
+		{
+			Name = propertyContents.Name.Name;
+			IsStatic = propertyContents.Getter.IsStatic;
+			AssociatedTypes.AssociatedTypeCollection.UnionWith (propertyContents.Getter.ReturnType.GetAssociatedTypes ());
+			Type = SwiftTypeToString.MapSwiftTypeToString (propertyContents.Getter.ReturnType, propertyContents.Class.ClassName.Module.Name);
+			GenericParameters = new DBGenericParameters (propertyContents.Getter.ReturnType);
+
+			Getter = new DBFunc (this, "Getter");
+			if (propertyContents.Setter != null)
+				Setter = new DBFunc (this, "Setter");
```

Comment:
Try to see if with ?? you can write nicer code (tip: you can)

Reply:
So I read up on the null coalescing operator and I'm having trouble applying it here.
I want to check the if propertyContents.Setter is null but I don't want to set as the value of Setter if it is not null.